src/main/java/org/broadinstitute/hellbender/engine/ProgressMeter.java

```diff
@@ -210,15 +230,17 @@ public void stop() {
         Utils.validate( !stopped, "the progress meter has been stopped already");
         this.stopped = true;
         currentTimeMs = timeFunction.getAsLong();
-        logger.info(String.format("Traversal complete. Processed %d total records in %.1f minutes.", numRecordsProcessed, elapsedTimeInMinutes()));
+        logger.info(String.format("Traversal complete. Processed %d total %s in %.1f minutes.", numRecordsProcessed, recordLabel, elapsedTimeInMinutes()));
```

Comment:
Thanks for taking into account my suggestions! :)

I'm using it also for a sorting iteration, but it is alright anyway.